<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="user-PC" failures="1" tests="1" name="Testexceldata" time="26.845" errors="0" timestamp="2021-05-14T18:15:55 IST">
  <testcase classname="testcases.Testexceldata" name="test1" time="10.109">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at testcases.Testexceldata.test1(Testexceldata.java:43)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
at java.base/java.lang.Thread.run(Thread.java:832)
... Removed 22 stack frames]]>
    </failure>
  </testcase> <!-- test1 -->
</testsuite> <!-- Testexceldata -->
